docs/book/src/plugins/declarative-v1.md

```diff
@@ -0,0 +1,37 @@
+# Declarative Plugin
+
+The declarative plugin is for building declarative cluster operators with kubebuilder.
+Declarative operators provide a fast path to orchestrating Kubernetes deployments to enable domain experts to focus
+on their component instead of re-answering questions like 'How do I get this YAML into the cluster?' or 'How do I
+update it?'.  
+With the declarative plugin manifests are created under `channels/packages/<packagename>/<version>/manifest.yaml` 
+and they will be loaded and applied to the cluster. An example of the pattern can be found [here](https://github.com/kubernetes-sigs/kubebuilder/tree/master/testdata/project-v3-addon)
+The `channels/stable` file declares the current stable version that will be used by default.
+
+## Advantages
+
+* Working with manifests lets us release a new app version without needing a new operator version
+* The declarative manifest makes it easier for users to understand what is changing in each version
+* It should result in less / simpler code
+
```

Comment:
WDYT, we added something like: 

## How to use:

### If you want that any API(s) and its respective controller(s) generate to reconcile them of your project adopt this partner then:
Use these plugin when you create a new project from the scratch, e.g:

```
kb init --plugins=declarative/v1
```

### If you want to adopt this partner for specific API(s) and its respective controller(s) generate to reconcile then: 
Use this plugin when you are creating the api

kb init create api  "todo: add e.g options from quick start" --plugins=declarative/v1